### PR TITLE
Path to project.json now shown

### PR DESCRIPTION
**Pull request by BrennanConroy**
"Error in C:\{path}\project.json" is put in front of the exception text now

@ChengTian 
#735

### CODE REVIEW

**Comment by troydai** [comment]
Should we use `Logger` instead? https://github.com/aspnet/XRE/blob/brecon/bug735/src/Microsoft.Framework.Runtime.Common/Impl/Logger.cs#L14

**Comment by anurse** [comment]
Or maybe put the string in the exception Message itself?

**Comment by ChengTian** [comment]
Yes, we should follow the original approach and put the string in exception message.

**Comment by BrennanConroy** [comment]
Made the FileFormatException insert the path into the exception string

**Comment by davidfowl** [comment]
What is the purpose of this change?

**Comment by BrennanConroy** [comment]
To tell the developer where the project.Json file that failed is located. The exception message doesn't contain it

Sent from my HTC

----- Reply message -----

**Comment by davidfowl** [comment]
Did you test this change in visual studio?

**Comment by davidfowl** [comment]
I think you should let the exception get thrown and handle it in the apphost just like compilation exceptions are handled

File: src/Microsoft.Framework.Runtime/FileFormatException.cs [hunk 1]
```diff
@@ -63,5 +63,10 @@ internal static FileFormatException Create(JsonReaderException exception, string
                 Line = exception.LineNumber
             };
         }
+
+        public override string ToString()
+        {
+            return String.Format("Error in {0}\n{1}", Path, base.ToString());
```
Comment (victorhurdugaci) [inline src/Microsoft.Framework.Runtime/FileFormatException.cs]: `Environment.NewLine` instead of `\n` ?

**Comment by BrennanConroy** [comment]
Running kpm restore/build propagates this exception to kre.host.shared where it calls PrintError(ex) so we can't handle it in apphost.

**Pull request closed without merging**